recipes/GoodVibes/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "goodvibes" %}
+{% set version = "3.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c7dfef22868f57762e910888e45daf96e2e9cc6cbb7cf299cc6a22cec37ee5c
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy
```

Comment:
I don't think `numpy` isn't required under `host` as it's not a build requirement and is only need to run the package

Reply:
Thanks for the help I've added these changes